aspnetcore/security/enforcing-ssl.md

```diff
@@ -1,37 +1,39 @@
 ---
-title: Enforcing SSL in an ASP.NET Core app
+title: Enforcing TLS in an ASP.NET Core app
 author: rick-anderson
-description: Shows how to require SSL in a ASP.NET Core web app
+description: Shows how to require HTTPS/TLS in a ASP.NET Core web app.
 manager: wpickett
 ms.author: riande
-ms.date: 07/19/2017
+ms.date: 2/9/2018
 ms.prod: asp.net-core
 ms.technology: aspnet
 ms.topic: article
 uid: security/enforcing-ssl
 ---
-# Enforcing SSL in an ASP.NET Core app
+# Enforcing HTTPS/TLS in an ASP.NET Core app
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This document shows how to:
 
-- Require SSL for all requests (HTTPS requests only).
+- Require TLS for all requests (HTTPS requests only).
 - Redirect all HTTP requests to HTTPS.
 
-## Require SSL
+> [!WARNING]
+> Do **not** use [RequireHttpsAttribute](/dotnet/api/Microsoft.AspNetCore.Mvc.RequireHttpsAttribute) on Web APIs that receive sensitive information. `RequireHttpsAttribute` does transparent redirection from HTTP to HTTPS, so the client never knows the information was sent insecurely over HTTP. Web APIs should close the connection with status code 400 (Bad Request) and not serve the request.
```

Comment:
not "transparent". Better would be "RequireHttpsAttribute" uses HTTP status codes to redirect browsers from HTTP to HTTPS which API clients may not understand or obey. Such clients may end up sending information over HTTP. Web APIs should either not listen on HTTP or should close the connection with status code 400 (Bad Request) and not serve the request.